int8 charging_state #0 = Charging, 1 = Half Full And Charging, 2 = Fully Charging
int16 kick_pwm
float32 cap_volt
